src/PKSim.UI/Views/PopulationAnalyses/BoxWhiskerChartView.cs

```diff
@@ -9,17 +14,45 @@ namespace PKSim.UI.Views.PopulationAnalyses
 {
    public partial class BoxWhiskerChartView : BaseChartView<BoxWhiskerXValue, BoxWhiskerYValue>, IBoxWhiskerChartView
    {
+      private BarItemLink _exportIndividualsMenu;
+
       public BoxWhiskerChartView(IImageListRetriever imageListRetriever, IToolTipCreator toolTipCreator) : base(imageListRetriever, toolTipCreator)
       {
          InitializeComponent();
          ChartsDataBinder = new BoxWhiskerChartDataBinder(this);
-         _chartControl.Legend.AlignmentHorizontal = LegendAlignmentHorizontal.Left;
-         _chartControl.Legend.AlignmentVertical = LegendAlignmentVertical.TopOutside;
+         Chart.Legend.AlignmentHorizontal = LegendAlignmentHorizontal.Left;
+         Chart.Legend.AlignmentVertical = LegendAlignmentVertical.TopOutside;
       }
 
       public void AttachPresenter(IBoxWhiskerChartPresenter presenter)
       {
          _presenter = presenter;
       }
+
+      protected override void ConfigurePopup(CancelEventArgs cancelEventArgs)
+      {
+         base.ConfigurePopup(cancelEventArgs);
+         _exportIndividualsMenu.Visible = canExtract;
+      }
+
+      public override void AddDynamicMenus(bool allowEdit = true)
+      {
+         base.AddDynamicMenus(allowEdit);
+         _exportIndividualsMenu = Chart.AddPopupMenu(PKSimConstants.MenuNames.ExtractIndividualByPercentile, extractIndividuals, ApplicationIcons.Individual, beginGroup: true);
+      }
+
+      private void extractIndividuals()
+      {
+         if (!canExtract)
+            return;
+
+         int index = _latestTrackedCurvedData.GetPointIndexForDisplayValues(_latestSeriesPoint.NumericalArgument, _latestSeriesPoint.Values[0]);
```

Comment:
This is how the actual box index is retrieved. (the CurvedData is a serie, and the latestSeriePoint is the point in the serie)